CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.4.1
+
+Release date: TBD
+
+### Bug Fixes
+
+#### macOS
```

Comment:
There is no technical difference, so just a naming. So we can choose by marketing reason.